.redhat-ci.yml

```diff
@@ -1,12 +1,9 @@
 host:
-    distro: fedora/24/cloud
-branches:
-    - master
+    distro: fedora/24/atomic
 packages:
```

Comment:
That won't work right now though AFAICS:

https://github.com/jlebon/redhat-ci/blob/master/main#L213